cloud/terraform/terraform_controller.py

```diff
@@ -1,13 +1,15 @@
 import os
 import json
 from typing import final
```

Comment:
I think we are not using this library in the code. The same for the next line.

Reply:
I don't even know where those came from